reana-ui/src/components/Table.js

```diff
@@ -0,0 +1,222 @@
+/*
+	-*- coding: utf-8 -*-
+
+	This file is part of REANA.
+	Copyright (C) 2018 CERN.
+
+	REANA is free software; you can redistribute it and/or modify it under the
+	terms of the GNU General Public License as published by the Free Software
+	Foundation; either version 2 of the License, or (at your option) any later
+	version.
+
+	REANA is distributed in the hope that it will be useful, but WITHOUT ANY
+	WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+	A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with REANA; if not, see <http://www.gnu.org/licenses>.
+
+	In applying this license, CERN does not waive the privileges and immunities
+	granted to it by virtue of its status as an Intergovernmental Organization or
+	submit itself to any jurisdiction.
+*/
+
+import React, { Component } from "react";
+import { Icon, Menu, Segment, Table } from "semantic-ui-react";
+import TableProgress from "./TableProgress";
+import TableActions from "./TableActions";
+import _ from "lodash";
+
+const URL = "http://reana-dev.cern.ch/api/workflows?";
```

Comment:
Do we need to have this hardcoded? If so, perhaps create `config.js` in the toplevel?